coc/events.py

```diff
@@ -349,6 +350,19 @@ async def wrapped(cached_war, war, callback):
 
         return _ValidateEvent.shortcut_register(wrapped, tags, custom_class, retry_interval, WarEvents.event_type)
 
+    @classmethod
+    def new_war(cls, tags=None, custom_class=None, retry_interval=None):
+        """Alias for the preparation start time changes, which is equal to a new war started"""
+
+        async def wrapped(cached_war: ClanWar, war: ClanWar, callback):
```

Comment:
This is LIT! Is there a test for this written?

Reply:
not yet